staff/staff_views_helper.py

```diff
@@ -134,44 +163,56 @@ def email_classroom(request, email_list, classroom_name):
     messages.add_message(request, messages.SUCCESS, "Email sent successfully")
 
 
-def get_emails_from_form(form):
-    email_list = []
-    groups = form.getlist("recipient_groups")
+def get_users_from_form(form):
+    user_list = []
+    groups = form.cleaned_data.get("recipient_groups")
     for group in groups:
-        group_name = Group.objects.get(id=group)
-        users = DjangoUser.objects.filter(groups__name=group_name)
+        users = DjangoUser.objects.filter(groups__name=group.name)
         for user in users:
-            email_list.append(user.email)
-    classrooms = form.getlist("recipient_classrooms")
+            user_list.append(user)
+    classrooms = form.cleaned_data.get("recipient_classrooms")
     for classroom in classrooms:
         classroom_object = Classroom.objects.get(id=classroom)
-        teacher = DjangoUser.objects.get(id=classroom_object.teacher_id).email
-        email_list.append(teacher.email)
+        teacher = DjangoUser.objects.get(id=classroom_object.teacher_id)
+        user_list.append(teacher)
         teacher_assistant = DjangoUser.objects.get(
             id=classroom_object.teacher_assistant_id
-        ).email
-        email_list.append(teacher_assistant.email)
+        )
+        user_list.append(teacher_assistant)
         students = DjangoUser.objects.filter(
             classroom_students__course=classroom_object.course
         )
         volunteers = DjangoUser.objects.filter(
             classroom_volunteers__course=classroom_object.course
         )
         for student in students:
-            email_list.append(student.email)
+            user_list.append(student)
         for volunteer in volunteers:
-            email_list.append(volunteer.email)
+            user_list.append(volunteer)
```

Comment:
the extend method could use used to make this a little shorter:

```python
user_list.extend(students)
user_list.extend(volunteers)
```

Reply:
Changed in 0cb86d82183a8324ddadf7471d8c308d6d9d2832 -- Thank you for pointing this method out, I will look for other places to use it, and will surely use it in the future.